lbz/lambda_to_lambda/invoke.py

```diff
@@ -0,0 +1,74 @@
+"""This file has an underscore at the end because just 'lambda' is a keyword in Python."""
+import json
+from typing import Any, Iterable, Optional, Type
+
+from lbz.aws_boto3 import client
+from lbz.exceptions import LambdaErrorResponse
+from lbz.lambda_to_lambda.enums import LambdaResult
+from lbz.misc import get_logger
+
+logger = get_logger(__name__)
+
+
+class SetsEncoder(json.JSONEncoder):
+    def default(self, o: Any) -> Any:
+        if isinstance(o, (set, frozenset)):
+            return list(o)
+        return json.JSONEncoder.default(self, o)
+
+
+class LambdaClient:
```

Comment:
I think it would be much easier to verify this code and think about improvements when you put here temporarily the current version (without improvements you applied) 😉